starlette/routing.py

```diff
@@ -591,27 +566,13 @@ def __init__(
         self.on_shutdown = [] if on_shutdown is None else list(on_shutdown)
 
         if lifespan is None:
-            self.lifespan_context: typing.Callable[
-                [typing.Any], typing.AsyncContextManager
-            ] = _DefaultLifespan(self)
-
-        elif inspect.isasyncgenfunction(lifespan):
```

Comment:
I took the opportunity to remove those... But I can readd them, and remove only on 1.0. Please ask me to readd.